### ISSUE TITLE
Update build.sh script to use new fabrikate semver semantics

### ISSUE THREAD

**Issue by evanlouie** [microsoft/bedrock#793]
**As a:** operator

**I want:**

the build.sh script to NOT prefix the the `$VERSION_TO_DOWNLOAD` link with `v`

OR

Use the go1.13 installation available on all
[ubuntu 16](https://github.com/microsoft/azure-pipelines-image-generation/blob/master/images/linux/Ubuntu1604-README.md)
and
[ubuntu 18](https://github.com/microsoft/azure-pipelines-image-generation/blob/master/images/linux/Ubuntu1804-README.md)
hosted vms and update all azure-pipelines.yaml which require `fab` to include:

```yaml
variables:
  GOROOT: "/usr/local/go1.13" # Go installation path
  GO111MODULE: "on"
```

and have just have `download_fab()` changed to:

```sh
function download_fab() {
  go get -v github.com/microsoft/fabrikate # automatically downloads the latest tagged release and installs the fabrikate binary as "fabrikate"
  alias fab=fabrikate # give "fabrikate" a "fab" alias to not break existing pipelines
}
```

**So that:** Fabrikate can start publishing proper golang spec'd semver releases
(which require `v` prefix) and those new releases won't break the build.sh
script

**Describe the solution you'd like:**

Either of the 2 mentioned above. The latter using `go get...` is simpler overall
and falls more inline with community semantics of how to do Golang releases.

**Acceptance Criteria:**

**Describe alternatives you've considered:**

**Additional context:**

**Does this require updates to documentation?:**

**Comment by andrebriggs** [comment]
@evanlouie is this still an issue?

**Comment by evanlouie** [comment]
Yes -- not required, but nice to have. Another issue that might want to add to this one is pinning the fabrikate download to a specific semver (i.e `0.x.x`) so we can unblock https://github.com/microsoft/fabrikate/pull/296.
